web/profiles/custom/yalesites_profile/modules/custom/ys_themes/config/install/ys_themes.component_overrides.yml

```diff
@@ -0,0 +1,77 @@
+callout:
+  field_style_color:
+    values:
+      blue-yale: 'Blue'
+      gray-700: 'Gray'
+      beige: 'Beige'
+    default: 'blue-yale'
+cta_banner:
+  field_style_color:
+    values:
+      gray-800: 'Dark Gray'
+      gray-700: 'Gray'
+      gray-200: 'Light Gray (gray-200)'
+      gray-100: 'Light Gray (gray-100)'
+      white: 'White'
+      blue-yale: 'Blue'
+    default: 'gray-800'
+  field_style_position:
+    values:
+      bottom: 'Bottom'
+      left:  'Left'
+      right: 'Right'
+    default: 'bottom'
+pull_quote:
+  field_style_variation:
+    values:
+      bar-left: 'Bar Left'
+      bar-right: 'Bar Right'
+      quote-left: 'Quote Left'
+    default: 'bar-left'
+text_with_image:
+  field_style_position:
+    values:
+      image-left: 'Image Left'
+      image-right: 'Image Right'
+    default: 'image-left'
+  field_style_width:
+    values:
+      feature: 'Feature'
+      highlight: 'Highlight'
+    default: 'feature'
+  field_style_variation:
+    values:
+      equal: 'Equal'
+      image: 'Image'
+      content: 'Content'
+    default: 'equal'
+quick_links:
+  field_style_variation:
+    values:
+      promotional: 'Promotional'
+      subtle: 'Subtle'
+    default: 'promotional'
+news_list:
+  field_style_variation:
+    values:
+      'true': 'Featured'
+      'false': 'Secondary'
+    default: 'true'
+news_cards:
+  field_style_variation:
+    values:
+      'true': 'Featured'
+      'false': 'Secondary'
+    default: 'true'
+event_list:
+  field_style_variation:
+    values:
+      'true': 'Featured'
+      'false': 'Secondary'
+    default: 'true'
+event_cards:
+  field_style_variation:
+    values:
+      'true': 'Featured'
+      'false': 'Secondary'
```

Comment:
I know that these are bool in storybook. I'm wondering if it should be. I'm thinking there could be other display modes for cards in in the future so this convention does not give us room to grow. I will start a conversation in Teams. Luckily this is the type of thing that can change later.

Reply:
Actually nevermind. We will push this as is. Brian recently suggested that complex card displays should use the future Views-Lite tool. So let's not overthink it at this stage.